validate.lic

```diff
@@ -678,6 +678,18 @@ class DRYamlValidator
             .any? { |skill| error("Skill name in cycle_armors_regalia is not valid: #{skill}") }
   end
 
+  def assert_that_non_training_spells_are_in_base_spells(settings)
+    return unless settings.offensive_spells || settings.buff_spells
+
+    spell_names = []
+    settings.buff_spells.each { |spell| spell_names.push(spell.first) }
```

Comment:
❓ What do you think about also checking the other 'spell' settings? I'm looking at https://github.com/rpherbig/dr-scripts/blob/master/dependency.lic#L858-L901 for an example

Reply:
I think that's a great idea for waggles, I just want to make sure I don't pull in khris and whatever.  Shouldn't be a problem.  I'm a little less sure about the need for it in all the training spells, since I don't even really think it harms anything there.  AFAIK crafting_training_spells shouldn't even care if the name is spelled wrong?

Reply:
No worries, I didn't have a chance to look at each setting in detail. These two are better than what we had before, so I'm fine with keeping just these two.

Reply:
No, I guess I'm thinking of it from the wrong angle.  If the name is screwed up then they probably won't have all the other stuff defined either and it won't pull from the defaults.  Yeah, I'll go back to the original and include everything.